tests/e2e/csi_snapshot_basic.go

```diff
@@ -4603,4 +4603,286 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 		}
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
+
+	/*
+	   Volume provision and snapshot creation/restore on VVOL Datastore
+	*/
```

Comment:
Can you add test steps in the comment section

Reply:
Added